docs/source/metrics.rst

```diff
@@ -72,12 +140,42 @@ Predefined Epoch Metrics
 .. autoclass:: SKLearnMetrics
     :members:
 
+
+TorchMetrics
+------------
+
+`TorchMetrics <https://torchmetrics.readthedocs.io/>`__ is a library implementing a lot of known metrics in PyTorch.
+They are passed to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>` using either the  ``epoch_metrics`` argument or the ``torch_metrics`` argument.
+When passed as epoch metric, they are only displayed at the end of the epoch.
+When passed using specialized ``torch_metrics`` argument, they are ddisplayed for each batch and at the end of the epoch.
+See `TorchMetrics documentation <https://torchmetrics.readthedocs.io/en/latest/references/modules.html>`__ for available torch metrics.
+
+.. warning:: When using this argument, the torch metrics are computed at each batch. This
+    can significantly slow down the computations depending on the metrics used. In such case, we advise to
+    use them as epoch metrics instead.
+
```

Comment:
```suggestion
.. warning:: When using this argument, the torch metrics are computed at each batch. This
    can significantly slow down the computations depending on the metrics used. In such cases, we advise using them as epoch metrics instead.

```

Reply:
Due to that warning, I think `model` should raise a warning if a torch metric is passed as a batch metric.

Reply:
Tu peux pas les passer comme batch metrics, juste epoch ou torch metrics.

Reply:
donc ça raise un erreur?

Reply:
L'argument en question ici, c'est `torch_metrics` et non `batch_metrics`.

Reply:
Tiens, j'ai pushé une modification à ce bout là.

Reply:
Good for me now.